magento/inventory
# #2851 : Return to Stock checkbox is applied for all products from order not only for selected

### Description (*)


### Fixed Issues (if relevant)

1. magento/inventory#2851: Return to Stock checkbox is applied for all products from order not only for selected

### Manual testing scenarios (*)

1. Login as customer
2. Add two simple products to cart
3. Place order
4. Create invoice and shipment
5. Check Quantity and Salable Quantity decrease by 1 for appropriate products
![image](https://user-images.githubusercontent.com/39480008/76620206-ab119000-6552-11ea-87c8-b94e2ea94880.png)

6. Click `Credit Memo` on order page in admin
7. Check `Return to Stock` only for one product
![image](https://user-images.githubusercontent.com/39480008/76620057-5706ab80-6552-11ea-9c25-f88f505cf668.png)
8. Click `Refund Offline`

### Expected Result
Only the product selected `Return to Stock` selected will update the qty
![image](https://user-images.githubusercontent.com/39480008/76620241-be246000-6552-11ea-80b2-8d99c8ba8a10.png)


### Contribution checklist (*)
